### PR TITLE
Update events

### PR DESCRIPTION
**Pull request by JonathanPMohan**

### CODE REVIEW

File: src/components/EventItem/EventItem.js [hunk 1]
```diff
@@ -17,6 +18,18 @@ class EventItem extends React.Component {
     deleteSingleEvent(event.id);
   }
 
+  editEvent = (e) => {
+    e.preventDefault();
+    const { passEventToEdit, event } = this.props;
+    passEventToEdit(event.id);
+  }
+
+  listingClick = (e) => {
```
Comment (morecallan) [inline src/components/EventItem/EventItem.js]: What is this function doing?

File: src/components/EventItem/EventItem.js [hunk 2]
```diff
@@ -37,7 +53,7 @@ class EventItem extends React.Component {
     };
 
     return (
-      <div className="eventItem text-center mx-auto">
+      <div className="eventItem text-center mx-auto" onClick={this.eventClick}>
```
Comment (morecallan) [inline src/components/EventItem/EventItem.js]: What is happening with this onClick here?

**Review by morecallan — COMMENTED** [review]
A few questions! Let’s talk if you have questions.

File: src/components/EventItem/EventItem.js (hunk 1 above)
Comment (JonathanPMohan) [reply src/components/EventItem/EventItem.js]: Actually.. It's not doing anything!  I think it eventually would be so you could click on the event and get the ID, right?  It's where I was trying to figure things out by using the Zillione project as an example.  So it's basically a dead function at the moment.

**Review by morecallan — COMMENTED** [review]
Thanks for the change! I'm going to go ahead and merge this but with the time remaining tonight, go through and comment the code with what is happening. I think it will be helpful for you to comment each function.

**Pull request merged**